### DISCUSSION TITLE
A variation of the 3x+1 Conjecture

### DISCUSSION THREAD

**Discussion by GRudolph** [pamelafox/recursive-visualizations#22] [Show and tell]
It looks like this does call trees, not just recursion?
https://www.recursionvisualizer.com/?function_definition=def%20r%28n%29%3A%0A%20%20%20return%20%28n%20%26%20%28~%28n%20-%201%29%29%29%0A%0Adef%20f%28x%29%3A%0A%20%20%20return%203*x%20%2B%20r%28x%29%0A%20%0Adef%20g%28x%29%3A%0A%20%20%20if%20r%28x%29%20%3D%3D%20x%3A%0A%20%20%20%20%20%20return%20x%0A%20%20%20return%20g%28f%28x%29%29&function_call=g%287%29

**Comment by pamelafox** [comment]
Good point, it's a call graph visualizer. Usually recursive call graphs are the most interesting ones, but that one is also quite interesting. That still seems to be a form of recursion, no? Thanks for sharing it!

**Comment by GRudolph** [comment]
Yes. This is also recursive. :)

George Rudolph
Professor and Dept. Chair
Computer Science
CS 520j
Utah Valley University
________________________________
